saleor/dashboard/product/views.py

```diff
@@ -2,66 +2,83 @@
 
 from django.contrib import messages
 from django.core.urlresolvers import reverse_lazy
-from django.db import transaction
 from django.shortcuts import get_object_or_404, redirect
 from django.template.response import TemplateResponse
 from django.utils.translation import ugettext_lazy as _
 from django.views.generic import DeleteView
 
-from ...product.models import Product
+from ...product.models import Product, ProductImage, Stock, ProductAttribute
 from ..utils import paginate
 from ..views import StaffMemberOnlyMixin, staff_member_required
-from .forms import (ProductImageFormSet, ProductClassForm, get_product_form,
-                    get_product_cls_by_name, get_variant_formset)
+from .forms import (ProductClassForm, get_product_form,
+                    get_product_cls_by_name, get_variant_form,
+                    ProductImageForm, get_verbose_name, StockForm,
+                    get_variant_cls, ProductAttributeForm,
+                    AttributeChoiceValueFormset, VariantAttributesForm)
 
 
 @staff_member_required
 def product_list(request):
-    products = Product.objects.prefetch_related('images').select_subclasses()
+    products = Product.objects.prefetch_related('images')
```

Comment:
Why do we drop `select_subclasses`? Can I no longer have more than one product class?